README.md

```diff
@@ -120,7 +120,8 @@ class Example extends React.Component {
                    onSuggestionsUpdateRequested={this.onSuggestionsUpdateRequested}
                    getSuggestionValue={getSuggestionValue}
                    renderSuggestion={renderSuggestion}
-                   inputProps={inputProps} />
+                   inputProps={inputProps}
+                    selectFirstSuggestion={false} />
```

Comment:
I think a lot of people go straight to the example code first so it's valuable to have properties listed there with the default values so that they're at least aware they exist. This is handled similarly in `redux-devtools` [here](https://github.com/gaearon/redux-devtools/commit/7f3cdf3b48b789984549bd862b41707b956f247a). Maybe there could/should be a comment.

Thoughts?